app/utils/validation.py

```diff
@@ -49,13 +49,11 @@ def validate_core_data_payload(payload):
     # test the input data
     if 'context' not in payload:
         raise ValueError("Payload requires 'context' field")
-    if 'states' not in payload or not payload['states']:
```

Comment:
This is kind of silly but what if we said: `if ('states' not in payload or not payload['states']) or payload['context']['dataEntryType'] == 'research':`

Reply:
Forcing states for "daily" and no states for "research"?
I was thinking to completely retiring "daily" maybe? but it sounds like a funny overkill